doc/UsersGuide/BuildingRunningTesting/BuildSRW.rst

```diff
@@ -361,6 +350,42 @@ The build will take a few minutes to complete. When it starts, a random number i
 
    If you see the ``build.out`` file, but there is no ``ufs-srweather-app/exec`` directory, wait a few more minutes for the build to complete.
 
+.. _install-uw:
+
+Install ``uwtools``
+^^^^^^^^^^^^^^^^^^^^
+
+The :uw:`UW Tools documentation <sections/user_guide/installation.html>` has the most up-to-date installation instructions. Users should refer to that documentation as authoritative. The UW team welcomes questions in its :uw-repo:`GitHub Discussions <discussions>` forum. 
+
+For convenience, a suggested procedure is included below for users who do not have ``uwtools`` or ``conda`` installed. However, in the event of problems, refer to the UW Tools documentation and forums. 
+
+#. Run ``uname -om`` to determine the system's operating system and architecture.
+#. Go to the `Miniforge releases page <https://github.com/conda-forge/miniforge/releases>`_ and download the desired version of Miniforge. For example:
+
+   .. code-block:: console
+      
+      wget https://github.com/conda-forge/miniforge/releases/download/24.11.2-1/Miniforge3-24.11.2-1-Linux-x86_64.sh
+
+#. Run the shell script to install ``conda``. For example:
+
+   .. code-block:: console
+
+      bash Miniforge3-24.11.2-1-Linux-x86_64.sh -bfp $PWD/conda 
+   
+   Users should replace ``Miniforge3-24.11.2-1-Linux-x86_64.sh`` with the name of the file they downloaded. 
+#. Remove the installation script, e.g., by running: ``rm Miniforge3-24.11.2-1-Linux-x86_64.sh``.
+#. Run: 
+
+   .. code-block:: console
+      
+      source conda/etc/profile.d/conda.sh
+      conda activate
+      cd ufs-srweather-app/conda/envs
+      conda create -n srw_app -c ufs-community -c conda-forge --override-channels uwtools=<X.Y.Z>
+   
+   where ``<X.Y.Z>`` is the desired version number. (It may be necessary to create the ``conda/envs`` directory within the ``ufs-srweather-app`` using the ``mkdir`` command if it does not already exist.)
+   Hit ``y`` to continue installation. 
+
 .. _MacLinuxDetails:
```

Comment:
Suggesting to remove Mac/Linux Details section, especially because is it very outdated and no longer correct for the updated ways it is built on MacOS systems

Reply:
On the other hand, if this is not planned to be merged into the srw-v3.0.0, the develop branch could still retain the MacOS/Linux docs in [develop] branch, to be updated when possible.
Otherwise many other locations where Linux/MacOS is mentioned need to be removed as well.

Reply:
The contents if this PR will also be merged into the release branch, but regardless, this PR only does what the PR Description says. Changes outside of that scope will happen in a separate PR. Josh is already planning to remove Linux/Mac info in one of his PRs (I think in a currently open one, but I'm not positive). It does not need to be done here.